pwa-devdocs/src/product-recs/index.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Integrating Product Recommendations
+---
+
+You can integrate Product Recommendations powered by [Adobe Sensei](https://www.adobe.com/sensei.html) into your PWA storefront.
+## Integration overview
+
+![Product Recommendations for PWA Studio](images/pwa-arch-diag-sensei.svg)
+
+Magento's [Product Recommendations powered by Adobe Sensei](https://docs.magento.com/user-guide/marketing/product-recommendations.html) is a feature backed by several SaaS services.
+The **Store** side includes your PWA storefront, which contains the event collector and recommendations layout template, and the backend, which includes the GraphQL endpoints, SaaS Export module, and the Admin UI.
+
+After you install the Product Recommendations PWA extension on your store, it will start sending [behavioral data](https://devdocs.magento.com/recommendations/events.html) to Adobe Sensei.
+Adobe Sensei processes this behavioral data along with the catalog data from the Magento backend and calculates the product associations leveraged by the recommendations service.
+At this point, the merchant can create and manage recommendation units from the Magento Admin UI then fetch those product recommendation units from their PWA storefront.
+
+## Install the Product Recommendations module
+
+Product Recommendations support on PWA requires installing the `venia-product-recommendations` module and the Product Recommendations Magento module.
+
+1. You can install the PWA `venia-product-recommendations` package from the NPM registry:
+
+   ```sh
+   npm install @magento/venia-product-recommendations
+   ```
+   This package contains everything you need to collect behavioral data and render the recommendations. Some recommendation types use behavioral data from your shoppers to train machine learning models that build personalized recommendations. Other recommendation types use catalog data only and do not use any behavioral data. See the [Magento user guide](https://docs.magento.com/user-guide/marketing/product-recommendations.html#trainmlmodels) to learn how Adobe Sensei trains machine learning models that results in higher quality recommendations.
+
+1. To install the Magento Product Recommendations backend module, see the [Magento developer documentation](https://devdocs.magento.com/recommendations/install-configure.html).
+This module expands Magento's existing GraphQL API to include fields that are used to fetch the recommendations from the Recommendations Service.
+
+## Create recommendation units
+
+Creating a product recommendation unit for your PWA storefront is similar to [creating one for a Magento storefront](https://docs.magento.com/user-guide/marketing/create-new-rec.html).
+The difference is that after you create a recommendation unit, you then need to add code to your PWA storefront to explicitly fetch the recommendation unit from the Magento backend. You can use one of the methods described below.
+
+## Fetch recommendations
+
```

Comment:
Add this, or something to the effect. 

Due to the limited scope of PWA-Studio's extensibility framework for extensions, this extension is unable to inject recommendations into the markup by itself. We recommend using the local intercept file that storefronts have available to inject Product Recommendations onto the page. Soon we will provide an example of how to inject product recommendations via the local intercept file, but for now please reference this existing [documentation](https://magento.github.io/pwa-studio/pwa-buildpack/extensibility-framework/#targetables-in-storefront-projects) on how to do it. Without injecting product recommendations in the markup, recommendations will not be fetched or rendered.